jipdate.py

```diff
@@ -229,16 +259,28 @@ def get_jira_issues(jira, username):
 
     f.write(msg)
     vprint("Found issue:")
+
+    unsorted_issues={}
     for issue in my_issues:
+        issue_id = "%s" % issue
+        this_issue = {}
+
+        this_issue["comments"] = ""
         vprint("%s : %s" % (issue, issue.fields.summary))
-        f.write("[%s]\n" % issue)
-        f.write("# Header: %s\n" % issue.fields.summary)
-        f.write("# Type: %s\n" % issue.fields.issuetype)
-        f.write("# Status: %s\n" % issue.fields.status)
-        f.write(get_extra_comments())
-        if last_comment:
-            write_last_jira_comment(f, jira, issue)
-        f.write("\n")
+        this_issue["comments"] += "[%s]\n" % issue
```

Comment:
So, some more thinking about this, it can be pretty complicated. What you want to achieve I guess is:
```
[Initiative1]
  [Epic1]
  [Epic2]
```

or
```
[Epic1]
  [Story1]
  [Story2]
```

or
```
[Initiative1]
  [Epic1]
    [Story1]
    [Story2]
  [Epic2]
    Story3]
    [Story4]
```
I.e, you want to show the tree. However, there are no limitations to how many parents you can have. One Epic can implement several Initiatives, likewise a Story can implement many Epics (this is not the normal case, but it can happen). So let's say an Epic implements two Initiatives, how should that be represented in text format?
```
[Initiative1, Initiative2]
  [Epic1]
    [Story1]
    [Story2]

[Initiative1]
  [Epic2]
    Story3]
    [Story4]
```
Take it one more level, let's say Story1 implements Epic1 and Epic2, then things are becoming pretty complicated when it comes to how to represent it :)

Just putting Stories in a group linking to a single Epic is easy, so is grouping Epics pointing to a single Initiative. But that will not cover the corner cases.